File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
    
    for i := 0; i < len(nums); i++ {
        for j := 0; j < len(nums); j++ {
            if nums[i] + nums[j] == target {
                if i != j {
                    return []int{i, j}
                }
            }
        }
    }

    return []int{}
}

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let check = {};

    for(let i = 0; i < nums.length; i++){
        const complement = target - nums[i];

        if(check[complement] !== undefined){
            return [check[complement], i];
        }

        check[nums[i]] = i;
    }

    return []
};